dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1584,20 +1585,18 @@ protected SearchHits indexSearch(String query, int limit, int offset, String sor
 
             searchRequest.source(searchSourceBuilder);
             response = RestHighLevelClientProvider.getInstance().getClient().search(searchRequest, RequestOptions.DEFAULT);
-
-
-        } catch (ElasticsearchStatusException | IndexNotFoundException | SearchPhaseExecutionException infe ) {
-            Logger.warn(this.getClass(), "----------------------------------------------");
-            Logger.warn(this.getClass(), "Elasticsearch Index Error : " + indexToHit);
-            Logger.warnAndDebug(this.getClass(), infe.getMessage(), infe);
-            Logger.warn(this.getClass(), "----------------------------------------------");
-
+        } catch (final ElasticsearchStatusException | IndexNotFoundException | SearchPhaseExecutionException e) {
+            Logger.error(this.getClass(), "----------------------------------------------");
+            Logger.error(this.getClass(), String.format("Elasticsearch error in index '%s'", indexToHit));
```

Comment:
are you sure that you wanna show these exceptions as error? they might cause a lot of noise. I think Logger.warnAndDebug is better

Reply:
Fixed